content/en/blog/2023-04-01-march-update/index.md

```diff
@@ -0,0 +1,256 @@
+  the resources applied by the kustomize-controller. This may improve memory usage on
+  large scale clusters at the cost of more direct API calls.
+- Changes to the logging of all controllers to ensure Kubernetes components like the
+  discovery client use the configured logging format.
+- New `flux events` command to display Kubernetes events for Flux resources, including
+  the events of a referenced resource.
+- Custom annotations can now be set with `flux push` using `--annotations`.
+
+New documentation
+
+- Cheatsheet: [Enable Helm drift
+  detection](/flux/cheatsheets/bootstrap/#enable-helm-drift-detection)
+- Cheatsheet: [Enable Helm near OOM
+  detection](/flux/cheatsheets/bootstrap/#enable-helm-near-oom-detection)
+- Cheatsheet: [Allow Helm DNS
+  lookups](/flux/cheatsheets/bootstrap/#allow-helm-dns-lookups)
+- Controller: [New helm-controller feature gates and
+  options](/flux/components/helm/options/#feature-gates)
+- Controller: [New kustomize-controller feature
+  gate](/flux/components/kustomize/options/#feature-gates)
+- Spec: [HelmRelease drift
+  detection](/flux/components/helm/helmreleases/#drift-detection)
+
+Big thanks to all the Flux contributors that helped us with this release!
+
+### Security news
+
+### Flagger x.y.z
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.19.0>
+
+#### Terraform-controller
+
+#### Flux Subsystem for Argo
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
+It's important to keep you up to date with new features and developments
+in Flux and provide simple ways to see our work in action and chat with
+our engineers.
+
+### Recent Events (ICYMI) 📺
+
+We feel blessed to have such a big community of users, contributors and
+integrators and so many are happy to talk about their experiences. In
+March here are a couple of talks we would like to highlight.
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+April - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+#### CloudNativeCon / KubeCon EU 2023
+
+...
+
+#### Flux Bug Scrub
+
```

Comment:
@kingdonb Anything you'd like to say here for March?

Reply:
I think this suggested edit didn't make it into the post last month:
https://github.com/fluxcd/website/pull/1365#discussion_r1112292225

I'd like to make a slightly more concrete announcement, but I haven't got any feedback except from one person. Maybe the blog post isn't the right place to solicit feedback about the timing for a Bug Scrub APAC...

Reply:
Ah, I must have missed that! Let's include it this time.